src/NzbDrone.Api/Frontend/Mappers/StaticResourceMapper.cs

```diff
@@ -27,6 +27,7 @@ public override bool CanHandle(string resourceUrl)
         {
             return resourceUrl.StartsWith("/Content") ||
                    resourceUrl.EndsWith(".js") ||
+                   resourceUrl.EndsWith(".map") ||
```

Comment:
.map is for the sourcemap right, or are we talking about a different kind of map? Isn't a sourcemap embedded in the .js better?